examples/sveltekit-openai/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "dev": "vite dev",
+    "dev": "vite dev --port 3000",
```

Comment:
did you mean to add this as part of this pr?

Reply:
i felt it was not worth separating out